.gitignore

```diff
@@ -0,0 +1,3 @@
+*.pyc
```

Comment:
Why would you ignore python cache files if there are absolutely no python files?

Reply:
Ah right. I didn't consider that.